babel.config.js

```diff
@@ -304,13 +304,30 @@ module.exports = function (api) {
     config.plugins.push("babel-plugin-istanbul");
   }
 
+  if (api.caller(caller => caller && caller.name == "@babel/eslint-parser")) {
+    // We disable all the plugins and presets, since we are only interested in parsing JS.
```

Comment:
Instead of caller detection in Babel config, can we disable the whole Babel config loading in 
https://github.com/babel/babel/blob/1516a42af21f65a4233eb7d66968da954882a162/eslint/babel-eslint-config-internal/index.js#L7 ?

See also https://github.com/babel/babel/tree/main/eslint/babel-eslint-parser#additional-parser-configuration